RecoLocalCalo/HGCalRecAlgos/src/RecHitTools.cc

```diff
@@ -420,10 +421,14 @@ bool RecHitTools::isSilicon(const DetId& id) const {
   bool issilicon = false;
   if (id.det() == DetId::HGCalEE || id.det() == DetId::HGCalHSi)
     issilicon = true;
+  if (id.det() == DetId::Forward && id.subdetId() == static_cast<int>(HFNose)) {
+    issilicon = true;
+  }
   return issilicon;
 }
 
 bool RecHitTools::isOnlySilicon(const unsigned int layer) const {
+  // HFnose TODO
```

Comment:
should this be done in this PR?

Reply:
for now the function is good, i.e. for the HFnose it results always true.
The bhLastLayer_ and bhOffset_ are not really meaningful for the HFnose as the hadronic section is not partitioned in FH and BH.
I let the comments to keep in mind later.